internal/handlers/pg/pgdb/pool.go

```diff
@@ -378,6 +376,17 @@ func (pgPool *Pool) EnsureTableExist(ctx context.Context, db, collection string)
 	return true, nil
 }
 
+// TableExists returns true if given FerretDB database / PostgreSQL schema
+// and FerretDB collection / PostgreSQL table exist.
+func (pgPool *Pool) TableExists(ctx context.Context, db, collection string) (bool, error) {
```

Comment:
I'd love to write some (integration) tests for such functions, but I see there are no tests in `handlers/pg` nor in `pgdb`...

Reply:
Let's do that